lib/Raven/Client.php

```diff
@@ -916,22 +570,19 @@ protected function get_user_data()
         ];
     }
 
-    protected function get_extra_data()
-    {
-        return $this->extra_data;
-    }
-
     public function get_default_data()
```

Comment:
I agree, but I did not change anything that was not tied to the purpose of this PR, which was moving the configuration properties to its own class